### ISSUE TITLE
Question about how to train across several datasets

### ISSUE THREAD

**Issue by Darren-greenhand** [Luodian/Otter#226]
Hi! We want to reproduce the training pipeline but we are not sure how you train otter on several datsets
Did you run the training code once for each dataset?  thus one time for LACONV, one time for LAi2i
And if so, did you reset the state of optimizer and scheduler every time?

thx !

**Comment by Darren-greenhand** [comment]
> i noticed in issue [#149 (comment)](https://github.com/Luodian/Otter/issues/149#issuecomment-1602517362)
> 
> the author seperated different dataset names with comma.

thank you！It helps a lot 😆
